README.md

```diff
@@ -46,21 +46,22 @@ This roster is seeded with approvers from the Client workgroups.
 
 ## Schedule
 
-| Release | Release Date | Serving        | Eventing        | PKG cut    | Unpin repos
-| ------- | ------------ | -------------- | --------------- | ---------- | -----------
-| v0.17   | 2020-08-18   | yanweiguo      | Harwayne        | 2020-08-11 | 2020-08-19
-| v0.18   | 2020-09-29   | ZhiminXiang    | n3wscott        | 2020-09-22 | 2020-09-30
-| v0.19   | 2020-11-10   | julz           | n3wscott        | 2020-11-03 | 2020-11-11
-| v0.20   | 2021-01-12   | nak3           | slinkydeveloper | 2021-01-07 | 2021-01-13
-| v0.21   | 2021-02-23   | mattmoor       | lionelvillard   | 2021-02-16 | 2021-02-24
-| v0.22   | 2021-04-06   | markusthoemmes | evankanderson   | 2021-03-30 | 2021-04-07
-| v0.23   | 2021-05-18   | tcnghia        | vaikas          | 2021-05-11 | 2021-05-19
-| v0.24   | 2021-06-29   | dprotaso       | matzew          | 2021-06-22 | 2021-06-30
-| v0.25   | 2021-08-10   | yanweiguo      | lionelvillard   | 2021-08-03 | 2021-08-11
-| v0.26   | 2021-09-21   | ZhiminXiang    | evankanderson   | 2021-09-14 | 2021-09-22
-| v1.0    | 2021-11-02   | julz           | bmo             | 2021-10-26 | 2021-11-03
-| v1.1    | 2021-12-14   | nak3           | matzew          | 2021-12-07 | 2021-12-15
-| v1.2    | 2022-01-25   | dprotaso       | psschwei        | 2022-01-18 | 2022-01-26
+| Release | Release Date | Serving        | Eventing            | PKG cut    | Unpin repos
+| ------- | ------------ | -------------- | --------------------| ---------- | -----------
+| v0.17   | 2020-08-18   | yanweiguo      | Harwayne            | 2020-08-11 | 2020-08-19
+| v0.18   | 2020-09-29   | ZhiminXiang    | n3wscott            | 2020-09-22 | 2020-09-30
+| v0.19   | 2020-11-10   | julz           | n3wscott            | 2020-11-03 | 2020-11-11
+| v0.20   | 2021-01-12   | nak3           | slinkydeveloper     | 2021-01-07 | 2021-01-13
+| v0.21   | 2021-02-23   | mattmoor       | lionelvillard       | 2021-02-16 | 2021-02-24
+| v0.22   | 2021-04-06   | markusthoemmes | evankanderson       | 2021-03-30 | 2021-04-07
+| v0.23   | 2021-05-18   | tcnghia        | vaikas              | 2021-05-11 | 2021-05-19
+| v0.24   | 2021-06-29   | dprotaso       | matzew              | 2021-06-22 | 2021-06-30
+| v0.25   | 2021-08-10   | yanweiguo      | lionelvillard       | 2021-08-03 | 2021-08-11
+| v0.26   | 2021-09-21   | ZhiminXiang    | evankanderson       | 2021-09-14 | 2021-09-22
+| v0.27   | 2021-11-02   | julz           | bmo                 | 2021-10-26 | 2021-11-03
```

Comment:
Let's not use version 0.x lets use the new version scheme 1.0 1.1 1.2 1.3 etc

Reply:
seems like a merge mistake